beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/handler/ExpiredMetadataHandler.java

```diff
@@ -58,6 +60,26 @@ public ExpiredMetadataHandler(CleanerClientFactory cleanerClientFactory,
     return housekeepingMetadataRepository.findRecordsForCleanupByModifiedTimestamp(instant, pageable);
   }
 
+  @Override
+  public void handleDisabledTable(String database, String table) {
```

Comment:
Shouldn't this function also take into account the `dryRunEnabled` flag like `cleanupMetadata()` ?

Reply:
oh yes, i'll add that